src/main.rs

```diff
@@ -239,12 +236,19 @@ alias z='pazi_cd'
         } else {
             // unwrap is safe because of the 'matches.len() == 0' check above.
             print!("{}", matches.last().unwrap().0);
-            process::exit(0);
         }
-    };
-
-    // By default print the frecency
-    for el in frecency.items_with_frecency() {
-        println!("{:.4}\t{}", el.1 * 100f64, el.0);
+    } else if flags.args.len() > 0 {
```

Comment:
This change has the side effect of making `z` not work with `PAZI_DEBUG=true`. Whoops!

I need to figure out the right way to express this mess in clap I think

Reply:
Fixed, silly me, forgot I had a positional arg